docker/README.md

```diff
@@ -0,0 +1,190 @@
+Open ONI In Docker
```

Comment:
I think it would be helpful to put a line or two in the actual README that users will see when they arrive pointing them towards these setup instructions.